app/src/main/res/layout/layout_grid_item.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Root view can be a material card view instead of a LinearLayout.

Do not forget to remove the paddings as well since item spacings are set by the decoration class.